vllm/core/scheduler.py

```diff
@@ -593,7 +596,7 @@ def _schedule_prefills(
         # Copy the queue so that the input queue is not modified.
         waiting_queue = deque([s for s in waiting_queue])
 
-        leftover_waiting_sequences = deque()
+        leftover_waiting_sequences: deque = deque()
```

Comment:
Is this type hint required?

Reply:
I think we should use https://docs.python.org/3/library/typing.html#typing.Deque

I don't know if we should suppress this issue, but I think it has an issue because you don't know the element type, and mypy cannot automatically infer that?

Reply:
Ah, my bad. The len and type annotation perhaps hint a weird python implementation. Also interesting that __bool__ is not supported.